src/components/ViewContainer/index.ts

```diff
@@ -208,7 +207,7 @@ class ViewContainer extends BridgeBase {
           //  component is a View (and not a ViewContainer). We now look for the
           //  "template" type directly in the created component (newComp) and NOT
           //  as a child of the newly created component.
-          if (this.bLogging) { console.log(`---> ${this.theComponentName} created new ${newCompName}`); }
+          if (true) { console.log(`---> ${this.theComponentName} created new ${newCompName}`); }
```

Comment:
No eslint disable of no-console  requied?

Reply:
Not really required until we do full linting. There are a **lot** of these in the code and I don't want to change them all now, we'll figure out the right/best way to improve this when we add full linting. But I fixed the other comment.